clean_data.py

```diff
@@ -91,7 +101,11 @@ def sii_from_audiogram(
     return sii.sii(ssl=ssl, nsl=nsl, hearing_threshold=critical_band_hl)
 
 
-def sii_from_df(df, ear='R'):
+def sii_from_df(df, ear: Literal['R', 'L']):
+    
+  """Extracts hearing thresholds from the specified frequencies, filters out the invalid data (NaN or infinite values), 
+  and computes SII only if there are 2 valid points, with default of any ear"""
```

Comment:
And what is the result if there are not 2 valid points?

Reply:
Yeah, it should be NaN